src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -130,20 +131,20 @@ ReportAie::writeReport(const xrt_core::device* /*_pDevice*/,
         _output << fmt8("%s") % "Link Register" % tile.second.get<std::string>("core.link_register");
         _output << fmt8("%s") % "Stack Pointer" % tile.second.get<std::string>("core.stack_pointer");
 
-	if(is_less) {
-	  _output << std::endl;
-	  continue;
-	}
+	      if(is_less) {
+	        _output << std::endl;
+	        continue;
+	      }
 
-	if(tile.second.find("dma") != tile.second.not_found()) {
+	      if(tile.second.find("dma") != tile.second.not_found()) {
```

Comment:
Spacing looks wrong

Reply:
No, I cross checked. Previous spacing was wrong, I corrected it